src/main/java/org/codehaus/mojo/license/LicenseUrlReplacement.java

```diff
@@ -0,0 +1,69 @@
+package org.codehaus.mojo.license;
+
+/*
+ * #%L
+ * License Maven Plugin
+ * %%
+ * Copyright (C) 2018 Codehaus
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+
+import org.apache.maven.plugins.annotations.Parameter;
+
+/**
+ * Defines a license url pattern and replacement pair.
+ *
+ * @since 1.17
+ */
+public class LicenseUrlReplacement
+{
+    /**
+     * Regular expression used to identify license urls that are to be replaced.
+     *
+     * @since 1.17
+     */
+    @Parameter
+    private String regexp;
+
+    /**
+     * Replacement license url.
```

Comment:
Plz add a piece of text informing the user that replacement is used in `java.util.regex.Matcher.replaceAll(String)`, esp. (citing from replaceAll(String) JavaDoc) `Note that backslashes (\) and dollar signs ($) in the replacement string may cause the results to be different than if it were being treated as a literal replacement string. Dollar signs may be treated as references to captured subsequences as described above, and backslashes are used to escape literal characters in the replacement string.`

Reply:
I believe this comment is addressed.